### PR TITLE
initial spree edge (1.1.0.rc1) version and 1-0-stable branch

### PR DESCRIPTION
**Pull request by wandtasie**
I've fixed some issues about rails 3.2.3 version with mass-assignment.
Now the master branch should work with spree edge, but not heavy tested
I'm able to:
assign ahot's to products
remove ahov's
change prices
add exclusions
and buy products with this possibilities in a default store...

### CODE REVIEW

File: Versionfile [hunk 1]
```diff
@@ -12,6 +12,7 @@
 #
 #
 #
-"0.60.x" => { :branch => "spree-0-60-stable" }
+"1.1.x" => { :branch => "master" }
+"1.0.x" => { :branch => "1-0-stable" }
```
Comment (wandtasie) [inline Versionfile:16]: @jsqu99 we need here the correct version of your new 1-0-stable branch

File: app/models/spree/product_decorator.rb [hunk 2]
```diff
@@ -14,7 +14,10 @@ module Spree
 
     def attach_option_values(ad_hoc_option_type)
       ad_hoc_option_type.option_type.option_values.each do |ov|
-        ad_hoc_option_type.ad_hoc_option_values << AdHocOptionValue.create(:option_value_id=>ov.id)
+        ahot = AdHocOptionValue.new()
```
Comment (wandtasie) [inline app/models/spree/product_decorator.rb:17]: I don't use mass assignment here cause vulnerability

**Comment by jsqu99** [comment]
thanks for all your work.  there's been a lot of activity in this extension in the last few days and i'm trying to review everything asap.

**Comment by wandtasie** [comment]
no problem, hope you enjoyed your holidays ;-)

**Pull request merged**